test/processors/concerns/http_files_test.rb

```diff
@@ -28,7 +28,7 @@ def logger
   it 'can download from http url' do
 
     stub_request(:get, http_uri.to_s).
-      with(headers: { 'Host' => 'test.prx.org:80' } ).
+      with(headers: { 'Host' => 'test.prx.org' } ).
```

Comment:
Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.